### DISCUSSION TITLE
Multiple overlayviews

### DISCUSSION THREAD

**Discussion by bn-piar** [JustFly1984/react-google-maps-api#3089] [Q&A] [answered]
Hey guys, I'm trying to add multiple overlayviews to the map but I'm getting `React.Children.only expected to receive a single React element child` error. 
```javascript
return (
  <Map {...mapOptions>
    {path.map((pos) => (
      <OverlayView key={pos.lat + pos.lng} position={pos} mapPaneName={OverlayView.FLOAT_PANE} />
    ))}
    <Polyline
      key={idx}
      options={{
        strokeColor: "#FF4C51",
        strokeOpacity: 1,
        strokeWeight: 6,
        path,
      }}
    />
  </Map>
);
```

I need multiple overlayviews to draw a route like this (using polylines and overlayviews as connecting points)

![image](https://user-images.githubusercontent.com/105789105/187197637-efc865bf-29e5-4c74-9599-fa33c15fa927.png)

How can I use multiple overlayviews? Maybe there's another way to draw what I need?

**Comment by JustFly1984** [comment]
Wrap return js in fragment

**Reply by bn-piar** [reply]
@JustFly1984 I did try that. Didn't work. Here's sandbox https://codesandbox.io/s/vibrant-aj-64do5r?file=/src/App.tsx

**Reply by bn-piar** [reply]
Should I create an issue @JustFly1984 ?

**Comment by bn-piar** [comment, marked as answer]
Okay so `OverlayView` requires a `children`.
```javascript
<OverlayView>
  <div />
</OverlayView>
```
